### PR TITLE
Inline edit basic info, update style, and add more Education types

### PR DESCRIPTION
**Pull request by BayLadyCoder**
Edit Basic info doesn't work with the old existing contacts that contain email type = null.

### CODE REVIEW

File: src/lib/formValidator.js [hunk 1]
```diff
@@ -112,7 +112,7 @@ const newProfileValidator = values => {
     err.lastName_error = 'Required';
   }
 
-  if (email === undefined) {
+  if (!email || email === undefined) {
```
Comment (dfkoh) [inline src/lib/formValidator.js]: This is redundant now, `!email` will always cover the case where `email === undefined` (but not vice versa). You could just have `!email`

File: src/pages/ProfilePage/ProfilePage.js [hunk 2]
```diff
@@ -62,7 +62,15 @@ const ProfilePage = ({
   inSelectMode,
 }) => {
   const scrollTo = useScroll();
+
   const [resumeLink, setResumeLink] = useState(null);
+  const [openForm, setOpenForm] = useState(false);
+
+  const handleUpdateContact = async values => {
+    await updateContact(values);
+    refreshContacts();
```
Comment (dfkoh) [inline src/pages/ProfilePage/ProfilePage.js]: You shouldn't have to call `refreshContacts` here as well, I don't think...`updateContact` should at the minimum handle the update

Comment (dfkoh) [reply src/pages/ProfilePage/ProfilePage.js]: I actually just realized why this call is probably here...I noticed you never updated the `reducer` to handle the `GET_CONTACT` or `UPDATE_CONTACT` result...that's actually a big problem. We should go over how to update that, but it should be a much better fix to this issue than adding `refreshContacts`

File: src/lib/formValidator.js (hunk 1 above)
Comment (BayLadyCoder) [reply src/lib/formValidator.js]: Done. (removed email === undefined)

File: src/pages/ProfilePage/ProfilePage.js (hunk 2 above)
Comment (BayLadyCoder) [reply src/pages/ProfilePage/ProfilePage.js]: added GET_CONTACT and UPDATE_CONTACT to reducer. Removed refreshContacts() in handleUpdateContact().

**Pull request merged**